item-impl/src/main/scala/com/example/auction/item/impl/ItemRepository.scala

```diff
@@ -50,6 +53,26 @@ private[impl] class ItemRepository(session: CassandraSession)(implicit ec: Execu
     }
   }
 
+  /**
+    * Motivation: https://discuss.lightbend.com/t/how-to-specify-pagination-for-select-query-read-side/870
+    */
+  private def selectItemsByCreatorInStatusWithPaging(creatorId: UUID, status: api.ItemStatus.Status, offset: Int, limit: Int) = {
+
+    implicit val system: ActorSystem = ActorSystem("ItemActorSystem")
+    implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+    val statement = new SimpleStatement(
+      """
+          SELECT * FROM itemSummaryByCreatorAndStatus
+          WHERE creatorId = ? AND status = ?
+          ORDER BY status ASC, itemId DESC
+          LIMIT ?
+        """, creatorId, status.toString, Integer.valueOf(limit)).setFetchSize(10)
```

Comment:
I think the comment in the method (`Motivation: https://discuss.lightbend.com/t/how-to-specify-pagination-for-select-query-read-side/870`) could give a little more context and make an explicit mention to the usage of `setFetchSize`.

`10` is a magic number. Seems a bit small (or maybe not, IDK). Could we make the page size dependant on the values of `offset` and `limit`? Hmmm, it probably doesn't make much sense and `10` is good enough. We could comment "`10` as a magic number fir the page size is OK because ???".

Reply:
Agreed, I'll replace the 10 by a constant with a proper name and a comment suggesting how to set the value depending on the application, including links to the datastax and lightbend's post.